content/telegraf/v1/install.md

```diff
@@ -396,47 +348,125 @@ The following commands are available:
 {{< /tab-content >}}
 {{< /tabs-wrapper >}}
 
+## Generate a custom configuration file
+
+The `telegraf config` command lets you generate a configuration file using Telegraf's list of plugins.
+
+### Create a configuration file with default input and output plugins
+
+To generate a configuration file with default input and output plugins enabled,
+enter the following command in your terminal:
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[Linux and macOS](#)
+[Windows](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```bash
+telegraf config > telegraf.conf
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```powershell
+.\telegraf.exe config > telegraf.conf
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+The generated file contains settings for all available plugins--some are enabled and the rest are commented out.
+
+### Create a configuration file with specific input and output plugins
+
+To generate a configuration file that contains settings for only specific input and output plugins, specify `telegraf` plugin filters--for example:
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[Linux and macOS](#)
+[Windows](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```bash
+telegraf --input-filter <pluginname>[:<pluginname>] --output-filter <outputname>[:<outputname>] config > telegraf.conf
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```powershell
+.\telegraf.exe --input-filter <pluginname>[:<pluginname>] --output-filter <outputname>[:<outputname>] config > telegraf.conf
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+For more advanced configuration details, see the
+[configuration documentation](/telegraf/v1/administration/configuration/).
+
 ## Custom-compile Telegraf 
 
 Use the Telegraf custom builder tool to compile Telegraf with only the plugins you need and reduce the Telegraf binary size.
 
-### Requirements
+1. [Prerequisites](#prerequisites)
+2. [Build the custom builder tool](#build-the-custom-builder-tool)
+3. [Run the custom builder to create a `telegraf` binary](#run-the-custom-builder-to-create-a-telegraf-binary)
 
-- Ensure you've installed [Go](https://go.dev/) version is 1.18.0 or later.
-- Create your Telegraf configuration file with the plugins you want to use. For details, see [Configuration options](/telegraf/v1/configuration/).
+### Prerequisites
 
-### Build and run the custom builder
+-  Install [Go](https://go.dev/) version 1.18.0 or later.
```

Comment:
We usually are on the latest Go version, so I would not say 18 or later. If they are behind, they won't be able to build as it will say something to the effect of "requested go version not found"

Reply:
> We usually are on the latest Go version, so I would not say 18 or later.

👍 Questioned that, but forgot to call it out.